spec/swagger.yaml

```diff
@@ -6586,6 +6590,232 @@ paths:
           source: |
             curl -H "Authorization: Bearer $smartlingToken" -H "Content-Type: application/json" https://api.smartling.com/vendors-api/v2/agencies/{agencyUid}/jobs
 
+#
+# Reports API
+#
+  '/reports-api/v3/word-count':
+    get:
+      description: >
+        Word Count report data for the specified parameters.
+      summary: Get Word Count Report
+      tags:
+      - Reports
+      parameters:
+      - name: startDate
+        in: query
+        type: string
+        required: true
+        format: YYYY-MM-DD
+        description: A parameter to limit words processed on or after ```startDate``` in UTC timezone.
```

Comment:
we have the America/New_York default TimeZone for Report Service.

Reply:
I don't think we can publish API that accepts NY timezone. We should rather change API to assume the date is in UTC

Reply:
A parameter to limit words processed on or after the ```startDate```, in UTC timezone.